### PR TITLE
Javadoc for Cipher

### PR DESCRIPTION
**Pull request by sundapeng**
A subtask of https://github.com/intel-hadoop/chimera/issues/91
Complete the javadoc for Cipher

### CODE REVIEW

File: src/main/java/com/intel/chimera/cipher/package-info.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Cipher classes
+ */
+package com.intel.chimera.cipher;
```
Comment (jerrychenhf) [inline src/main/java/com/intel/chimera/cipher/package-info.java]: Do we need a newline at the end?

**Comment by jerrychenhf** [comment]
Thanks @sundapeng contributing the java doc for cipher part.

File: src/main/java/com/intel/chimera/cipher/package-info.java (hunk 1 above)
Comment (sundapeng) [inline src/main/java/com/intel/chimera/cipher/package-info.java]: Yes, I have added the newline, thank you for your comment

**Comment by jerrychenhf** [comment]
Looks good to me.

**Comment by sundapeng** [comment]
Thank @jerrychenhf for your review.

**Pull request merged**